adp/v_adp.workers_custom_field_group.sql

```diff
@@ -38,7 +38,7 @@ SELECT w.associate_oid
       ,NULL AS bit_value
       ,CASE WHEN ISNUMERIC(cfg.codeValue) = 1 THEN CONVERT(INT, cfg.codeValue) END AS numeric_value
       ,CONVERT(NVARCHAR(128), JSON_VALUE(cfg.nameCode, '$.codeValue')) AS code_value
-      ,CONVERT(NVARCHAR(128), JSON_VALUE(cfg.nameCode, '$.shortName')) AS short_name
+      ,CONVERT(NVARCHAR(128), JSON_VALUE(cfg.nameCode, '$.codeValue')) AS short_name
```

Comment:
@kevinverhoff qq: is this a mistake or does the object not have a shortName attribute?

Reply:
Not a mistake - The object doesn't have a shortname